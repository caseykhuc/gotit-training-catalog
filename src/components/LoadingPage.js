import React from 'react';

const LoadingPage = () => (
  <div className="d-flex align-items-center justify-content-center" style={{ height: '20rem' }}>
    <div className="spinner-border" style={{ width: '3rem', height: '3rem' }} role="status">
      <span className="sr-only">Loading...</span>
    </div>
  </div>
)

export default LoadingPage;
