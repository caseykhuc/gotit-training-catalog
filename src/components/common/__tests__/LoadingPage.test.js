import React from 'react';
import { shallow } from 'enzyme';
import LoadingPage from '../LoadingPage';

describe('component/LoadingPage', () => {
  let props;
  let wrapper;

  const update = () => {
    wrapper.update();
  };
  const setup = () => {
    wrapper = shallow(<LoadingPage {...props} />);
    update();
  };
  beforeEach(() => {
    props = {
    }
  });
  it('should render correctly', () => {
    setup();
    expect(wrapper).toMatchSnapshot();
  });
})
