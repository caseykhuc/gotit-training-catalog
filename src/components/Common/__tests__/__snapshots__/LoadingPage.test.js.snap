// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component/Common/LoadingPage should render correctly 1`] = `ShallowWrapper {}`;
